sdk/sdktypes/project.go

```diff
@@ -47,3 +47,13 @@ func (p Project) WithNewID() Project { return p.WithID(NewProjectID()) }
 func (p Project) WithID(id ProjectID) Project {
 	return Project{p.forceUpdate(func(pb *ProjectPB) { pb.ProjectId = id.String() })}
 }
+
+type (
```

Comment:
I guess you don't want to do this in the style of all other types here. I can live with it for now since it's quite simple.